### ISSUE TITLE
[BUG] Importing a new pilot populates from custom Items

### ISSUE THREAD

**Issue by msprijatelj** [Eranziel/foundryvtt-lancer#499]
**Describe the bug**
Importing a new pilot pulls from the Items list instead of creating a fresh copy from the Compendium. This applies to the pilot's own items (like skills and talents) and their mech's items (like frames and systems).

**To Reproduce**
1. Create a pilot on CompCon. Give them the "Charm" Trigger.
2. Export the pilot's data in JSON (or share code, or save to a compcon account).
3. In Foundry, import a copy of the "Charm" Skill from the Skill compendium.
4. Edit the description imported "Charm" skill in a way that will stand out.
5. Create a new pilot sheet on Lancer for Foundry.
6. Import the pilot created in step 1 into the pilot sheet from step 3 with your method of choice.
7. Once import is complete, observe that the pilot's "Charm" trigger uses the description created in step 4.

**Expected behavior**
Importing a pilot retrieves a fresh copy of all the pilot's items from the Compendium.

**Platform (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox, Foundry App
 - Foundry version v9.280, v10.291
 - Lancer system version v1.3.1, v1.4.0-beta.1

**Additional context**
I see logs with the following message in the console:
```
LANCER | New pilot_weapon has data provided from an import, skipping default init.
```
This seems to indicate that this behavior is intended. If so, could someone please explain the use case to me? No use in having a bug report for intended behavior.

**Discord contact info**
Valkyrion#6196

**Comment by Eranziel** [comment]
I believe this was intended, the logic being that if you had a custom version in your world Items you would want it to override.

**Comment by msprijatelj** [comment]
Got it; if the functionality is still desirable then, feel free to mark this as Wontfix then
